setup/ConfigEditor.xaml.cs

```diff
@@ -0,0 +1,126 @@
+﻿using Microsoft.Win32;
+using Microsoft.WindowsAPICodePack.Dialogs;
+using System.ComponentModel;
+using System.IO;
+using System.Windows;
+
+namespace ReShade.Setup
+{
+	/// <summary>
+	/// Interaction logic for ConfigEditor.xaml
+	/// </summary>
+	public partial class ConfigEditor : Window
+	{
+		private const string INI_SECTION = "GENERAL";
+
+		private string _configFilePath;
+
+		public ConfigEditor(string configPath)
+		{
+			InitializeComponent();
+			_configFilePath = configPath;
+
+			_load();
+			Closing += (object sender, CancelEventArgs args) => _save();
+		}
+
+		private void _writeValue(string key, string value)
```

Comment:
methods in .NET should use PascalCasing for their names. starting them with '_' is not really great. Perhaps a thing to look into next time ;)